pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -143,20 +143,19 @@ def _do_subject_slice_timing(subject_data, TR, TA=None, spm_dir=None,
 
     # compute nslices
     nslices = load_vols(subject_data.func[0])[0].shape[2]
-    assert 1 <= ref_slice <= nslices, ref_slice
+#    assert 1 <= ref_slice <= nslices, ref_slice
 
-    # compute slice indices / order
-    if not isinstance(slice_order, basestring):
-        slice_order = np.array(slice_order) - 1
-    slice_order = get_slice_indices(nslices, slice_order=slice_order,
-                                    interleaved=interleaved)
+#    # compute slice indices / order
+#    if not isinstance(slice_order, basestring):
+#        slice_order = np.array(slice_order) - 1
+#    slice_order = get_slice_indices(nslices, slice_order=slice_order,
+#                                    inter  leaved=interleaved)
```

Comment:
The get_slice_indices function has been fixed for the bug you spotted, so I'd advice using it. It's useful if slice_order == "interleaved", in fact it does general sanitization. See comments on line 197 below.